src/App.js

```diff
@@ -1,15 +1,94 @@
 import React, { Component } from 'react'
 import ListArticles from './Components/ListArticles'
+import LoginForm from './Components/LoginForm'
+import { signInUser } from './state/actions/reduxTokenAuthConfig'
+import { connect } from 'react-redux'
 
 class App extends Component {
+  state = {
+    renderLoginForm: false,
+    email: '',
+    password: '',
+    getArticlesIndex: true
+  }
+
+  renderArticleGetState = () => {
+    this.setState ({
+      getArticlesIndex: false
+    })
+  }
+
+  renderFormState = () => {
+    this.setState({
+      renderLoginForm: !this.state.renderLoginForm
+    })
+  }
+
+  inputChangeHandler = (e) => {
+    this.setState({
+      [e.target.name]: e.target.value
+    })
+  }
+
+  handleLogin = () => {
+    const { signInUser } = this.props;
+    const { email, password } = this.state;
+      signInUser({ email, password })
+        .then(
+          console.log('Yaaaaaaay')
+        )
+        .catch(error => {
+          console.log(error)
+        })
+  }
 
   render() {
+    let renderLogin
+    let welcomeMessage
+
+    if (this.props.currentUser.isSignedIn) {
+      welcomeMessage = <p id="welcome-message">Hello {this.props.currentUser.attributes.email}</p>
+    }
+
+    if (this.state.renderLoginForm === true) {
+      renderLogin = (
+        <>
+          <LoginForm 
+            inputChangeHandlerProps={this.inputChangeHandler}
+            handleLoginProps={this.handleLogin}
+          />
+        </>
+      )
+    } else {
+      renderLogin = (
+        <button id="login-button" onClick={this.renderFormState}>Login</button>
+      )}
+
     return (
       <>
-        <ListArticles />
+        {renderLogin}
+        {welcomeMessage}
+
+        <ListArticles 
```

Comment:
Don't need this anymore, right? Please revert back to pre-state with just <ListArticles />

Reply:
Same as above.